dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -120,6 +120,20 @@ def get_tos_version
     render json: @user.terms_of_service_version.nil? ? -1 : @user.terms_of_service_version.inspect
   end
 
+  # GET /api/v1/users/current/permissions
+  def get_current_permissions
+    prevent_caching
+    if current_user
+      render json: {
+        permissions: current_user.permissions.map(&:permission)
+      }
+    else
+      render json: {
+        permissions: []
+      }
+    end
+  end
```

Comment:
since you only really care about levelbuilder permissions here, why check all of the permissions? I don't have an issue with this approach, just curious!

Reply:
I figured I would make a generic route rather than a levelbuilder-specific one.